website/docs/r/cis_waf_group.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "ibm"
+page_title: "IBM: cis_waf_group"
+sidebar_current: "docs-ibm-resource-cis-waf-group"
+description: |-
+  Provides an IBM Cloud Internet Services WAF Rule Group resource.
```

Comment:
UI Names it as CIS Rule Set.. Can this be refered as alternate name in the docs.

Reply:
I followed cli document.

Reply:
`CIS Rule Set` can this be teramed as alternate name in docs.. just to avoid confusion.. I myself took lil time to match btw cli and UI..So I feel it is better to added one more statement here

Reply:
can you please specify what is the statement?